testResources/json/course.json

```diff
@@ -0,0 +1,58 @@
+{
+    "api_version": 7,
+    "course": {
+        "id": 3,
+        "name": "test-course",
+        "details_url": "https://example.com/staging/courses/3.json",
+        "unlock_url": "https://example.com/staging/courses/3/unlock.json",
+        "reviews_url": "https://example.com/staging/courses/3/reviews.json",
+        "comet_url": "http://example.com:8443/comet",
+        "spyware_urls": ["https://example.com/staging"],
+        "unlockables": [],
+        "exercises": [
+            {
+                "id": 284,
```

Comment:
Nit: the #id doesn't really matter, but i guess we could start the numbering from 1...